source/_components/media_player.bluesound.markdown

```diff
@@ -46,3 +46,37 @@ media_player:
         port: 11000
       - host: 192.168.1.131
 ```
+
+### {% linkable_title Service `bluesound_join` %}
+
+Group players together under a single master speaker. That will make a new group or join to exists group.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `master` | no | A single `entity_id` that will become/hold the master speaker.
+| `entity_id` | no | String or list of a single `entity_id` that will group to master speaker.
+
+### {% linkable_title Service `bluesound_unjoin` %}
+
+Remove one or more speakers from a group of speakers. If no `entity_id` is provided, all speakers are unjoined.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id` | yes | String or list of `entity_id`s that will be separated from their master speaker.
+
+### {% linkable_title Service `bluesound_set_sleep_timer` %}
+
+Sets a timer that will turn off the speaker. For each time you call this it will increase the time by on step. The steps are (in minutes): 15, 30, 45, 60, 90, 0.
+If you increse an ongoing timer of for example 13 minutes, it will increase it to 15. If the timer is set to 90 it will remove the time (hence the 0).
```

Comment:
- `increse` -> `increase``
- Missing comma: `to 90 it will` -> `to 90, it will`

Reply:
Thank you, all three is changed